gapic-generator-java-pom-parent/pom.xml

```diff
@@ -27,7 +27,7 @@
         consistent across modules in this repository -->
     <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
     <grpc.version>1.55.1</grpc.version>
-    <google.auth.version>1.18.0</google.auth.version>
+    <google.auth.version>1.19.0</google.auth.version>
```

Comment:
We can revert this now that we have a separate deps PR for it.

Reply:
reverted and merged `main` into this PR